test/bin/build_images.sh

```diff
@@ -25,6 +25,31 @@ osbuild_logs() {
     done
 }
 
+extract_container_images() {
+    local -r version=$1
+    local -r repo_url=$2
+    local -r outfile=$3
+    local -r repo_name="microshift-extract-images"
+
+    echo "Extracting images from ${version}"
+    mkdir -p "${IMAGEDIR}/release-info-rpms"
+    pushd "${IMAGEDIR}/release-info-rpms"
+    sudo bash -c "cat > /etc/yum.repos.d/${repo_name}.repo" << EOF
+[${repo_name}]
+name="microshift ${version}"
+baseurl="${repo_url}"
+enabled=1
+check_gpg=false
+EOF
+    sudo dnf download microshift-release-info-"${version}" --repo "${repo_name}"
+    images="$(get_container_images "${version}" "${IMAGEDIR}/release-info-rpms")"
+    echo "${images}" >> "${outfile}"
+    sed -i 's/,/\n/g' "${outfile}"
```

Comment:
```suggestion
    echo "${images}" | sed 's/,/\n/g' >> "${outfile}"
```

Reply:
Why not go further and do: `get_container_images "${version}" "${IMAGEDIR}/release-info-rpms" | sed 's/,/\n/g' >> "${outfile}`

Reply:
Done